api/v1/swagger.yaml

```diff
@@ -11559,6 +11585,10 @@ definitions:
         readOnly: true
         description: >
           The collector id.
+      intervalSecs:
```

Comment:
Do we need to add this field to GeneratorFieldsParam, GeneratorResponse and HeartbeatGeneratorResponse ? I added currentCollector to those.